CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/translator/DatetimeAttributeTranslator.java

```diff
@@ -112,6 +115,10 @@ public String translate(final String value, final PluginParameters parameters) {
                     dateTime = df.parse(value);
                     break;
                 }
+                case EXCEL: {
+                    dateTime = TemporalFormatting.zonedDateTimeFromLong(DateUtil.getJavaDate(Double.parseDouble(value)).getTime());
+                    break;
+                }
```

Comment:
I know it's there in a couple other cases but if you're able to remove the curly braces (they're not needed), that would be great

Reply:
@antares1470, When I debugged, the epoch time for above example is -2209023000000 
(https://www.epochconverter.com/) produces both days below, so looks like all dates are converted to GMT. Should we display it in local time zone?
![image](https://user-images.githubusercontent.com/64826928/158520945-c2cdc133-d0c0-487d-a7b8-a30cd0d7d110.png)

Reply:
Right I guess that makes sense. If converting the excel time to epoch still produces the same value in Constellation and then I suppose for now it makes sense to keep as is. I suppose from my perspective it just looks a little weird inputing a time into a spreadsheet and then it importing as a different time in Constellation. Maybe what I will do though is put in a new issue to give an option to specify a time zone when applying the formatter (excel or epoch)